### DISCUSSION TITLE
Is there anyway to make camera to look at (x,y,z) position and normal of model

### DISCUSSION THREAD

**Discussion by balrajg** [google/model-viewer#2257] [Q&A]
### Description

Hi, Is there a way to move the camera to look at a position and normal of the model,

So i can fix the default viewing position for each and every model.

Example, I have to view other side of model by default, or rotate the model so other user can see other side 

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
You can do it with a combination of `camera-target` and `camera-orbit`, but you may need to do a little math to convert a normal vector to spherical coordinates: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-attributes-cameraOrbit
